packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -18,6 +18,7 @@ import {
 
 const { log } = out("Saml Client");
 const timeout = 120000;
```

Comment:
nit: since we're talking about `dayjs.duration` could also use it here since we're touching this file.